backend/src/db/providers/clientCourseProvider.ts

```diff
@@ -155,7 +155,7 @@ const updateClientCourseField = async (
 
 const arrangeAssessment = async (courseId: string) => {
   const updatedCourse = await ClientCourseModel.findByIdAndUpdate(courseId, {
-    $set: { withAssessment: true },
+    $set: { withAssessment: true, status: CourseStatus.assessment },
```

Comment:
NOOOOOOOOOOOO!

Pls, look carefully what are you changing. This change will set course status to `assessment` when approving course request with interview!!!